src/app/logged/student/student.component.ts

```diff
@@ -4,7 +4,6 @@ import { StudentService } from '../../services/student.service';
 import { AddComponent } from './add/add.component';
```

Comment:
Name of the import not explicit, could be redundant. StudentAddComponent could be a better choice.